tools/sof-dump-status.py

```diff
@@ -100,7 +100,7 @@ def _loadACard(self):
             card_info['id'] = output[idx].strip().split(' ')[0]
             card_info['short'] = output[idx].split(' - ')[-1].strip()
             card_info['type']=output[idx].strip().split()[1][1:]
-            card_info['desc']=output[idx + 1].strip()
+            card_info['longname']=output[idx + 1].strip()
```

Comment:
Do you mean the old description is actually the longname?
I remember there is some requirement for alsalib to support longname? Do we need to check the alsalib here first?

Reply:
OK. I will check it later